docs/tutorial/kymographs.rst

```diff
@@ -98,6 +98,15 @@ crop the region from `9.5` micron to `26` microns using the following command::
 
 .. image:: figures/kymographs/kymo_cropped.png
 
+If we know the bead diameter, we can also do this automatically using :meth:`~lumicks.pylake.kymo.Kymo.crop_beads()`.
```

Comment:
I think it's good anytime talking about this functionality we reinforce that it's an estimate and not guaranteed to be 100% accurate in all cases
```suggestion
If we know the bead diameter, we can automatically crop the kymo to an estimate of the bead edges using :meth:`~lumicks.pylake.kymo.Kymo.crop_beads()`.
```